.gitignore

```diff
@@ -100,7 +100,7 @@ web_modules/
 # TypeScript cache
 *.tsbuildinfo
 
-# Optional npm cache directory
+# Optional npm cache
```

Comment:
Not much point in tweaking comments in generated code, that will just be overwritten if someone re-gennerates this section